weave-js/src/components/PagePanel.tsx

```diff
@@ -120,6 +120,24 @@ function useEnablePageAnalytics() {
   const pathRef = useRef('');
   const {urlPrefixed, backendWeaveViewerUrl} = getConfig();
 
+  const trackOnPathDiff = useCallback(
+    (location: any, options: any) => {
+      const currentPath = `${location.pathname}${location.search}`;
+      const fullURL = `${window.location.protocol}//${window.location.host}${location.pathname}${location.search}${location.hash}`;
+      if (pathRef.current !== currentPath) {
+        let pageName = '';
+        if (location.search.includes('exp=get')) {
+          pageName = 'WeaveBoardOrTable';
```

Comment:
it would be more accurate to say `WeaveGetExpression` here - this could in theory point to any object type, not just a board or a table

Reply:
changed